python/ql/src/Security/CWE-090/LdapInjection.ql

```diff
@@ -5,13 +5,13 @@
  * @kind path-problem
  * @problem.severity error
```

Comment:
We need additional metadata. I think `@precision high` is fair, but we need to confirm this.

```suggestion
 * @problem.severity error
 * @security-severity 9.8
 * @precision high
```

`@security-severity 9.8` match [Java](https://github.com/github/codeql/blob/main/java/ql/src/Security/CWE/CWE-090/LdapInjection.ql) and [C#](https://github.com/github/codeql/blob/main/csharp/ql/src/Security%20Features/CWE-090/LDAPInjection.ql) version.

Reply:
Thanks for reminding me! It originally got the maximal false positive score, so `@precision high` should be correct :-)